src/m365/teams/commands/tab/tab-get.ts

```diff
@@ -0,0 +1,291 @@
+import commands from '../../commands';
+import GlobalOptions from '../../../../GlobalOptions';
+import {
+  CommandOption, CommandValidate
+} from '../../../../Command';
+import GraphCommand from '../../../base/GraphCommand';
+import Utils from '../../../../Utils';
+import request from '../../../../request';
+import { Team } from '../../Team';
+import { Channel } from '../../Channel';
+import { Tab } from '../../Tab';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  teamId: string;
+  teamName: string;
+  channelId: string;
+  channelName: string;
+  tabId: string;
+  tabName: string;
+}
+
+class TeamsTabGetCommand extends GraphCommand {
+  public get name(): string {
+    return `${commands.TEAMS_TAB_GET}`;
+  }
+
+  public get description(): string {
+    return 'Gets information about the specified Microsoft Teams tab';
+  }
+
+  public getTelemetryProperties(args: CommandArgs): any {
+    const telemetryProps: any = super.getTelemetryProperties(args);
+    telemetryProps.teamId = args.options.teamId;
+    telemetryProps.teamName = args.options.teamName;
+    telemetryProps.channelId = args.options.channelId;
+    telemetryProps.channelName = args.options.channelName;
+    telemetryProps.tabId = args.options.tabId;
+    telemetryProps.tabName = args.options.tabName;
+    return telemetryProps;
+  }
+
+  public getTeamId(teamName: string): Promise<string> {
+    if (Utils.isValidGuid(teamName as string)) {
```

Comment:
I know the intent behind this, but because of the argument name it looks confusing. Also, it seems very much like duplication of something that belongs in the `validate` method